lib/go/test/flow_dkg_test.go

```diff
@@ -830,4 +829,34 @@ func TestDKGNil(t *testing.T) {
 		assert.Equal(t, cadence.NewBool(false), result)
 
 	})
+
+	dkgKey1 := fmt.Sprintf("%0192d", access)
+
+	t.Run("Should count as completed even if 50 of participants sent nil keys", func(t *testing.T) {
```

Comment:
```suggestion
	// NOTE: one participant submitted a nil key vector in the previous test case
	t.Run("Should count as completed even if 50% of participants sent nil keys", func(t *testing.T) {
```
note about reliance on state from another test case.